### ISSUE TITLE
Notify user when an SVG file is edited and resaved as PNG

### ISSUE THREAD

**Issue by seafiish** [don-tnowe/godot-sprite-painter#8]
Since it's possible to open and draw on files that you can't actually edit (e.g. [SVG](https://github.com/don-tnowe/godot-sprite-painter/issues/2#issuecomment-1515510565)), there should either be:
1) a clear warning put up when you open such a file in the editor,
2) all the tools should be grayed out/locked,
3) or being able to open such files should be entirely removed.

I'd be happy to implement this myself if you're willing to bear with the time it will take me (as I'm new to Godot), just let me know which you prefer or if you have other ideas (also which filetypes that don't work).

**Comment by don-tnowe** [comment]
Actually, it is possible to open them. Kinda. Just saves it as `.png` in the same folder when you try (*thanks for reporting last time, that's what made me fix this*)

At one point, I thought of implementing proper SVG editing support. Simple raster editors are good for pixelart but some people despise it, or just prefer vector after working in Flash (*yes there are still people making games in Flash*) Also would mean I get to not touch Inkscape again.

But that would take lots of time because of a completely different toolset needed, plus programming in the performant SVG rendering

**Comment by FivelFox** [comment]
Oh! Good to know, perhaps we should document this and / or add a note when in svg files?

**Comment by don-tnowe** [comment]
Likely. I'll need to think how to better inform the user.

**Comment by DearFox** [comment]
I just accidentally lost a file because the plugin overwrote a png with the same name when opening an svg.
 Even though I didn't save it.  I think it would be nice to have something for file security.

**Comment by DearFox** [comment]
> I just accidentally lost a file because the plugin overwrote a png with the same name when opening an svg. Even though I didn't save it. I think it would be nice to have something for file security.

Yes, when you click on the Discard button, the plugin still saves the file, apparently.
